docs/operate/get-started/other-hardware/_index.md

```diff
@@ -788,6 +789,7 @@ On your machine's **CONFIGURE** tab in the [Viam app](https://app.viam.com), cli
 Select **Local module**, then **Local module**.
 
 Enter the absolute path to the <file>/bin/&#60;module-name&#62;</file> executable, for example `/home/jessamy/my-go-module/bin/mymodule` on Linux, or `/Users/jessamy/my-go-module/bin/mymodule` on macOS.
+For modules configured this way, `viam-server` uses this path instead of the `entrypoint` field in your <file>meta.json</file> file.
```

Comment:
This is a bit of a gotcha that may trip people up I expect - can you get this into the UI? Possibly when adding the local module OR on local module cards as a note of sorts.

Reply:
Okay! https://viam.atlassian.net/browse/APP-8332